<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>910</width>
    <height>704</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="personal_photo">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>80</y>
      <width>241</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Фото на личное дело</string>
    </property>
   </widget>
   <widget class="QPushButton" name="paper_photo">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>150</y>
      <width>241</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Паспорт</string>
    </property>
   </widget>
   <widget class="QPushButton" name="agree_photo">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>220</y>
      <width>241</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Согласие на ОПД</string>
    </property>
   </widget>
   <widget class="QPushButton" name="tabel_photo">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>290</y>
      <width>241</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Аттестат</string>
    </property>
   </widget>
   <widget class="QPushButton" name="achives_photo">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>360</y>
      <width>281</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Индивидуальные достижения</string>
    </property>
   </widget>
   <widget class="QPushButton" name="join_agree_photo">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>420</y>
      <width>261</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Согласие на зачисление</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
